shared/unattended/RHEL-6-perf-series.ks

```diff
@@ -0,0 +1,72 @@
+install
+KVM_TEST_MEDIUM
+text
+poweroff
+lang en_US.UTF-8
+keyboard us
+key --skip
+network --bootproto dhcp
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc Asia/Shanghai
```

Comment:
Is it okay for an non-Chinese user? And the further question is do we need a functionality to set this value dynamically (i.e. get the configuration from host and pass to here)?